crates/e2e/src/setup/colocation.rs

```diff
@@ -124,3 +124,70 @@ mempool = "public"
         driver::run(args.into_iter(), None).await;
     })
 }
+
+pub fn start_driver_with_zeroex_liquidity(
```

Comment:
Can't this be made part of the default flow or somehow? I'd like us to avoid adding a new setup method for each test case.

Reply:
Refactored https://github.com/cowprotocol/services/pull/2253/commits/dd49f3dba9fc25dbd557f8a6e87fb22279841dba

Reply:
> I believe eventually we need to redo how we setup the binaries to run to use a builder that takes care of all this.

That is definitely a good task for a separate PR.